cluster/config-defaults.yaml

```diff
@@ -947,3 +950,14 @@ sysctl_settings: ""
 
 # enables/disables the minDomains field for pod topology spread.
 min_domains_in_pod_topology_spread_enabled: "true"
+
+# Set defaults values to be used by skipper ingress when the open policy agent is enabled
+{{if eq .Cluster.Environment "test"}}
```

Comment:
Let's put this block of config items next to the related one `skipper_open_policy_agent_enabled`